server/src/modules/users/application/use-cases/protected-user/protected-user-controller.ts

```diff
@@ -0,0 +1,41 @@
+import express from 'express'
+import { ControllerWithDTO } from '../../../../../shared/app/controller-with-dto'
+import { ProtectedUserUseCase } from './protected-user-use-case'
+import { ProtectedUserDTO, protectedUserDTOSchema } from './protected-user-dto'
+import { Result } from '../../../../../shared/core/result'
+import { ValidationError } from 'joi'
+
+export class ProtectedUserController extends ControllerWithDTO<ProtectedUserUseCase> {
+  constructor(useCase: ProtectedUserUseCase) { super(useCase) }
```

Comment:
The name of the use case should have some sort of verb? What's the usecase doing? `GetProtectedUserUseCase`?